package.json

```diff
@@ -100,6 +100,7 @@
     ]
   },
   "devDependencies": {
+    "@types/reach__router": "^1.3.5",
```

Comment:
Did this get auto-added? Seems like it shouldn't be necessary as it's added [here](https://github.com/mozilla/fxa/pull/6131/files#diff-6f49eaae21bfa6a37a17d0c80d18d5bdR64) and only used within fxa-settings.